generic_config_updater/patch_sorter.py

```diff
@@ -544,6 +544,66 @@ def _get_default_value_from_settings(self, parent_path, field_name):
 
         return None
 
+class LaneReplacementMoveValidator:
```

Comment:
>LaneReplacementMoveValidator [](http://example.com/codeflow?start=6&length=28)

Checking the implementation, I do not see the relationship of "lane replacement", I am also thinking it could be generalized and not limited to PORT table.